Number_types/include/CGAL/Mpzf.h

```diff
@@ -66,7 +66,7 @@
 
 #include <boost/cstdint.hpp>
 
-#ifdef _MSC_VER
+#ifdef _MSC_VER && (! defined(__clang__))
```

Comment:
We should use [`BOOST_MSVC` from `<boost/config.hpp>`](https://www.boost.org/doc/libs/master/libs/config/doc/html/boost_config/boost_macro_reference.html#boost_config.boost_macro_reference.boost_informational_macros):

> `BOOST_MSVC`
> Defined if the compiler is really Microsoft Visual C++, as opposed to one of the many other compilers that also define `_MSC_VER`. Has the same value as `_MSC_VER`.

Reply:
Just found this:   `Installation/include/CGAL/auto_link/auto_link.h:#  define BOOST_MSVC _MSC_VER`